### DISCUSSION TITLE
Please link to the search page from main page

### DISCUSSION THREAD

**Discussion by markosjal** [iptv-org/community#513] [Ideas]
I know there is here somewhere a page that allows you to search for a specific channel, or at least there used to be. Why is that page not linked to from the main page?

**Comment by freearhey** [comment]
Added a link to the [site](https://iptv-org.github.io/) in the repository description.

<img width="264" alt="image" src="https://github.com/iptv-org/iptv-org.github.io/assets/7253922/f270e486-4870-4b03-9a99-6f918db8b61e">
